ia/src/game.py

```
@@ -0,0 +1,45 @@
+def cleanup_thread() -> None :
+    global threads
+
+    alive_thread = [t for t in threads if t.is_alive()]
+    threads = alive_thread
+
+def game(process_args) -> None :
+  global threads
+
+  server_adress = (process_args.machine, process_args.port)
+
+  unused_slot = 1;
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)